src/renderer/scss/component/_search.scss

```diff
@@ -1,106 +1,47 @@
-.wunderbar {
-  cursor: text;
-  display: flex;
-  flex: 1;
-  min-width: 175px;
-  position: relative;
-  z-index: 1;
-
-  > .icon {
-    top: 10px;
-    left: 10px;
-
-    position: absolute;
-    z-index: 1;
+.search__header {
+  background-color: $lbry-black;
+  color: $lbry-white;
+  padding: 2rem;
+
+  html[data-theme='dark'] & {
+    background-color: transparent;
+    border-bottom: 1px solid rgba($lbry-white, 0.1);
   }
 }
 
-.wunderbar__active-suggestion {
-  background-color: $lbry-blue-2;
-  color: $lbry-black;
-}
-
-.wunderbar__input {
-  width: 100%;
-  height: var(--btn-height);
-
+.search__title {
   align-items: center;
-  border-bottom: none; // ?
+  cursor: default;
   display: flex;
-  font-size: 13px;
-  justify-content: center;
-  padding: 10px 10px 10px 30px;
-  transition: background-color 0.2s;
+  font-weight: 700;
+  line-height: 1.33;
+  margin-bottom: 2rem;
 
-  &:not(:focus) {
-    background-color: rgba($lbry-gray-1, 0.3);
+  > * {
+    margin-left: 0.5rem;
   }
 
-  &:focus {
-    background-color: rgba($lbry-gray-1, 0.5);
+  @media (min-width: 601px) {
+    font-size: 2rem;
   }
-}
 
-.wunderbar__menu {
-  background-color: $lbry-white;
-  box-shadow: 0 1px 5px rgba($lbry-black, 0.15);
-  max-width: 100px; // ?
-  overflow: hidden;
-}
-
-.wunderbar__suggestion {
-  align-items: center;
-  cursor: pointer;
-  display: flex;
-  flex-direction: row;
-  justify-items: flex-start;
-  padding: 5px 10px;
-
-  &:not(:first-of-type) {
-    border-top: 1px solid transparent;
-  }
-
-  .icon {
-    flex-shrink: 0;
+  @media (max-width: 600px) {
+    font-size: 1.75rem;
   }
 }
 
-.wunderbar__suggestion-label {
-  overflow: hidden;
-  padding-left: $spacing-vertical;
-  text-overflow: ellipsis;
-  white-space: nowrap;
-}
 
-.wunderbar__suggestion-label--action {
-  font-size: 12px;
-  font-weight: 600;
-  line-height: 0.1; // to vertically align because the font size is smaller
-  margin-left: 0.5rem;
-  opacity: 0.7;
-  text-transform: uppercase;
-  white-space: nowrap;
-}
 
-.search__content {
-  padding: $spacing-width;
+.search__results-wrapper {
+  padding: 2rem;
 }
 
-.search__results {
-  margin-top: -$spacing-width;
-  padding-bottom: $spacing-vertical;
-  flex-flow: wrap column;
-
-  .search-result__row {
-    padding: $spacing-vertical 0;
-
-    &:first-of-type {
-      padding-top: 0;
-    }
-  }
+.search__results-section {
+  margin-bottom: 4rem;
```

Comment:
I don't think everything having the same margin makes sense.

Reply:
They don't need to have the same margins, but some factor of the base level marign. If we just had small, medium, and large margins, where small is 1/3 the base margin, medium is 1/2 and large is the base margin (these are just random numbers), it would greatly help with consistency.

If everything is on it's own, every component could have different margins if future changes aren't tracked carefully after this is merged.